src/QuadForm/Torsion.jl

```diff
@@ -1761,13 +1786,10 @@ function genus(T::TorQuadModule, signature_pair::Tuple{Int, Int})
   if length(sym2) <= 2 || sym2[3][1] != 2
     sym2 = insert!(sym2, 3, Int[2, 0, 1, 0, 0])
   end
-  if modulus_quadratic_form(T) == 1
-    # in this case the blocks of scales 1, 2, 4 are under determined
-    # make sure the first 3 symbols are of scales 1, 2, 4
-    # i.e. their valuations are 0, 1, 2
-
-    # the form is odd
-    block0 = [b for b in _blocks(sym2[1]) if b[4] == 1]
+  if modulus_quadratic_form(T) == 1 || parity == 1
+    _o = mod(parity, 2)
```

Comment:
???

Reply:
If `parity == 1`, I want 1 for the next line, and if `parity == 2`, I want 0.